src/pages/gig-detail-page.html

```diff
@@ -19,18 +20,32 @@
 
     .container {
       @apply(--app-desktop-width);
+      min-height: 320px;
+    }
+
+    .spinner {
+      text-align: center;
+      padding: 128px 0;
+    }
+
+    paper-spinner {
+      width: 64px;
+      height: 64px;
+      margin: 0 auto;
     }
   </style>
 
   <app-route route="{{route}}"
     pattern="/:gigId" data="{{routeData}}"></app-route>
   <data-gig-detail gig-id="[[gigId]]" gig="{{gig}}"></data-gig-detail>
 
-  <!-- TODO(zjones) Add spinner for RAIL -->
-
   <div class="container">
 
-    <div class="gig-detail-container">
+    <div class="spinner" hidden$="[[!loading]]">
```

Comment:
Does this bind to the `hidden` property and if so doesn't that mean it will still take up visible space on the page?